zerver/lib/test_classes.py

```diff
@@ -273,6 +273,12 @@ def users_subscribed_to_stream(self, stream_name, realm):
 
         return [subscription.user_profile for subscription in subscriptions]
 
+    def assertUrlServesContentsOfFile(self, url, result):
```

Comment:
@rafidaslam Sorry, the comment I wrote back in July shouldn't be taken too literally.  @TomaszKolek is correct here--let's use underscore notation.